eta/core/objects.py

```diff
@@ -108,6 +108,7 @@ def __init__(
             score: (optional) an optional score for the object
             frame_number: (optional) the frame number in which this object was
                 detected
+            index_in_frame: (optional) the object index in one frame
```

Comment:
clarify this a bit: `the index of this object in the frame where it was detected`

Reply:
done